SpatialGDK/Source/SpatialGDK/Private/Interop/WellKnownEntitySystem.cpp

```diff
@@ -130,4 +135,51 @@ void WellKnownEntitySystem::InitializeVirtualWorkerTranslationManager()
 	VirtualWorkerTranslationManager = MakeUnique<SpatialVirtualWorkerTranslationManager>(Receiver, Connection, VirtualWorkerTranslator);
 	VirtualWorkerTranslationManager->SetNumberOfVirtualWorkers(NumberOfWorkers);
 }
+
+void WellKnownEntitySystem::MaybeClaimSnapshotPartition()
+{
+	// Perform a naive leader election where we wait for the correct number of server workers to be present in the deployment, and then
+	// whichever server has the lowest server worker entity ID becomes the leader and claims the snapshot partition.
+	const Worker_EntityId LocalServerWorkerEntityId = GlobalStateManager->GetLocalServerWorkerEntityId();
+
+	if (LocalServerWorkerEntityId == SpatialConstants::INVALID_ENTITY_ID)
+	{
+		UE_LOG(LogWellKnownEntitySystem, Warning, TEXT("MaybeClaimSnapshotPartition aborted due to lack of local server worker entity"));
```

Comment:
Is this a warning? Can't this just happen because you checked out something else first? If not and it implies something has gone wrong should it be an error?

Reply:
You should only have interest in these entities once the server's worker entity has been created - can switch to an error if you want

Reply:
Unless it's correctly tagged that sounds like it's depending on specific query structures. This doesn't actually look like it breaks anything though, so if someone added some query on some other entity created at startup that wanted to know about these then this being an error would be confusing.

So if it doesn't break I'd argue it possibly shouldn't be more than a log (if logged at all).

Reply:
Discussed offline - keeping as a warning